clients/tabby-chat-panel/src/index.ts

```diff
@@ -33,7 +34,7 @@ export interface ChatMessage {
 }
 
 export function createClient(target: HTMLIFrameElement) {
-  return createThreadFromIframe(target)
+  return createThreadFromIframe(target) as Api
```

Comment:
So that the client instance can use the method correctly, e.g., client.init(), client.sendMessage().